code/game/objects/items/emags.dm

```diff
@@ -16,7 +16,8 @@
 	worn_icon_state = "emag"
 	var/prox_check = TRUE //If the emag requires you to be in range
 	var/type_blacklist //List of types that require a specialized emag
-
+	var/pshoom = 'sound/items/pshoom.ogg'
+	var/pshoom_2 = 'sound/items/pshoom_2.ogg'
```

Comment:
These vars seem useless, I'd just inline them

Reply:
fair point, will do

Reply:
What do you mean by this?